pkg/scheduler/core/generic_scheduler.go

```diff
@@ -279,6 +285,7 @@ func (g *genericScheduler) findNodesThatPassFilters(ctx context.Context, fwk fra
 	// Create feasible list with enough space to avoid growing it
 	// and allow assigning.
 	feasibleNodes := make([]*v1.Node, numNodesToFind)
+	errorStatuses := make([]*FilterStatus, numNodesToFind)
```

Comment:
not `errorStatuses` but `notFeasibleStatuses` or `unschedulableStatuses`, but more importantly, the number could be len(allNodes), not just `numNodesToFind`, and so it is not clear to me that allocating this array for all cases is worth the optimization.